xmtp_db/src/encrypted_store/group_intent.rs

```diff
@@ -73,6 +73,7 @@ pub struct StoredGroupIntent {
     pub staged_commit: Option<Vec<u8>>,
     pub published_in_epoch: Option<i64>,
     pub should_push: bool,
+    pub sequence_id: Option<i64>,
```

Comment:
will every intent type have a sequence_id? 

If so, where does the sequence id come from for each intent type

if not, which intents have a sequenceId and which don't